docs/content/docs/connectors/table/formats/avro.md

```diff
@@ -80,6 +80,14 @@ Format Options
       <td>String</td>
       <td>Specify what format to use, here should be <code>'avro'</code>.</td>
     </tr>
+    <tr>
+      <td><h5>avro.encoding</h5></td>
+      <td>optional</td>
+      <td>yes</td>
+      <td>binary</td>
+      <td>String</td>
+      <td>Serialization encoding to use. The valid enumerations are: binary, json. <a href="https://avro.apache.org/docs/current/specification/#encodings">(reference)</a></td>
+    </tr>
```

Comment:
Thanks for binging this topic up again. Since json format is slow, I would suggest adding more info here to describe the pros and cons and pointed out binary format is commonly recommended. Binary is the default setting, not only because of the backward compatibility, but also because of the performance. WDYT?

Reply:
That seems reasonable to me - I've added an additional explanation.